ambry-replication/src/main/java/com/github/ambry/replication/ReplicationManager.java

```diff
@@ -281,6 +281,20 @@ private List<RemoteReplicaInfo> createRemoteReplicaInfos(List<? extends ReplicaI
     return remoteReplicaInfos;
   }
 
+  public ReplicaThread getReplicationThread(String threadName) {
```

Comment:
I don't see who is using this method.

Reply:
spillover from next pr, will remove

Reply:
actually, its used in the ctor for BackupIntegMonitor